implementation/contracts/deposit/DepositLiquidation.sol

```diff
@@ -256,12 +256,14 @@ library DepositLiquidation {
         }
         if (contractEthBalance > 1) {
             if (_wasFraud) {
-                initiator.transfer(contractEthBalance);
+                /* solium-disable-next-line security/no-send */
+                initiator.send(contractEthBalance);
             } else {
                 // There will always be a liquidation initiator.
                 uint256 split = contractEthBalance.div(2);
                 _d.pushFundsToKeepGroup(split);
-                initiator.transfer(split);
+                /* solium-disable-next-line security/no-send */
+                initiator.send(address(this).balance);
```

Comment:
No loss of functionality here so I'm good, but a couple of questions:
- Do we want to leave space for the initiator to pull their funds manually in case this fails? This could use a raw `.call()` since the initiator would be paying for their withdrawal.
- Do we want to globally disable the security/no-send rule?

No need for further action here until we figure out how we want to tackle the inherent issues with the initiator split, though.

Reply:
As @Shadowfiend noted, the method could also bit split in to parts: (1) first part liquidating the deposit and pushing funds to keep (secure) (2) a method that allows the initiator to pull the deposit contracts balance. this would avoid a case where eth is locked in the contract because `send` fails (which should be unlikely).